### DISCUSSION TITLE
Is this real?  Error: .git (and subdirectories) are not executable

### DISCUSSION THREAD

**Discussion by hess8** [LycheeOrg/Lychee#2426] [Q&A] [answered]
I see this in diagnostics.  Is this really an error?  Lychee seems to be running OK. 

And if I do chmod -R +x .git, it doesn't remove the error message.

**Comment by ildyria** [comment]
> I see this in diagnostics. Is this really an error? Lychee seems to be running OK.
> 
> And if I do chmod -R +x .git, it doesn't remove the error message.

Lychee will be running fine, however it won't be able to check for updates.
`chmod -R +x .git` is only modifying for current user. Not for the web user.:
do `ls -lola` to see the full rights.

**Comment by hess8** [comment, marked as answer]
Thanks.  I did ` sudo chmod -R +x .git`, and it didn't help.  Then I did
` chown -R www-data:www-data .git`  and that fixed it.

`config, index and ORIG_HEAD `were owned by root, not `www-data`.   You might
want to broaden the error message to include ownership.

**Reply by ildyria** [reply]
not necessarily.
You could also have done `sudo chmod -R 775 .git` and that would also have fixed it.
